test/baseline.test.js

```diff
@@ -0,0 +1,40 @@
+const percySnapshot = require('@percy/webdriverio');
+var minimal_constants = require('../helper/minimal.constants');
+var common_constants = require('../helper/common.constants');
+var baseline_constants = require('../helper/baseline.constants');
+
+console.log('BASELINE_URL')
+console.log(minimal_constants.BASELINE_URL); 
+console.log(baseline_constants.BASELINE_NAME); 
+
+describe('Percy Baseline', () => {
```

Comment:
[percy] Percy has started!
[percy] Running "wdio run wdio.conf.js --spec test/baseline.test.js"

Execution of 1 workers started at 2022-10-27T00:31:12.268Z

[0-0] BASELINE_URL
[0-0] https://www.browserstack.com
[0-0] Browserstack Production Website Baseline
[0-0] RUNNING in firefox - /test/baseline.test.js
[percy] Snapshot taken: Homepage Page
[percy] Snapshot taken: Pricing Page
[0-0] Error in "Percy Baseline snapshot integrations"
Error: Timeout - Async function did not complete within 10000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
    at <Jasmine>
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
[0-0] Error in "Percy Baseline snapshot docs"
Error: Timeout - Async function did not complete within 10000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
    at <Jasmine>
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
[0-0] FAILED in firefox - /test/baseline.test.js

 "spec" Reporter:
------------------------------------------------------------------
[firefox 103.0.2 mac #0-0] Running: firefox (v103.0.2) on mac
[firefox 103.0.2 mac #0-0] Session ID: 35b0e43c-3766-4708-a8ef-be5a96090532
[firefox 103.0.2 mac #0-0]
[firefox 103.0.2 mac #0-0] » /test/baseline.test.js
[firefox 103.0.2 mac #0-0] Percy Baseline
[firefox 103.0.2 mac #0-0]    ✓ snapshot homepage
[firefox 103.0.2 mac #0-0]    ✓ snapshot pricing
[firefox 103.0.2 mac #0-0]    ✖ snapshot integrations
[firefox 103.0.2 mac #0-0]    ✖ snapshot docs
[firefox 103.0.2 mac #0-0]
[firefox 103.0.2 mac #0-0] 2 passing (59.6s)
[firefox 103.0.2 mac #0-0] 2 failing
[firefox 103.0.2 mac #0-0]
[firefox 103.0.2 mac #0-0] 1) Percy Baseline snapshot integrations
[firefox 103.0.2 mac #0-0] Error: Timeout - Async function did not complete within 10000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
[firefox 103.0.2 mac #0-0] Error: Timeout - Async function did not complete within 10000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
[firefox 103.0.2 mac #0-0]     at <Jasmine>
[firefox 103.0.2 mac #0-0]     at listOnTimeout (node:internal/timers:559:17)
[firefox 103.0.2 mac #0-0]     at processTimers (node:internal/timers:502:7)
[firefox 103.0.2 mac #0-0]
[firefox 103.0.2 mac #0-0] 2) Percy Baseline snapshot docs
[firefox 103.0.2 mac #0-0] Error: Timeout - Async function did not complete within 10000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
[firefox 103.0.2 mac #0-0] Error: Timeout - Async function did not complete within 10000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
[firefox 103.0.2 mac #0-0]     at <Jasmine>
[firefox 103.0.2 mac #0-0]     at listOnTimeout (node:internal/timers:559:17)
[firefox 103.0.2 mac #0-0]     at processTimers (node:internal/timers:502:7)


Spec Files:      0 passed, 1 failed, 1 total (100% completed) in 00:01:03 

Looks like your default test runner timeout is set at 10 seconds and some tests were not able to be completed in that time. Consider increasing timeout limit.